### ISSUE TITLE
HTTPS for typedoc.org

### ISSUE THREAD

**Issue by lgarron** [TypeStrong/typedoc#814]
Could you enable HTTPS per https://blog.github.com/2018-05-01-github-pages-custom-domains-https/ ? 😃

(Moved from https://github.com/Microsoft/tsdoc/issues/35#issuecomment-404619600)

**Comment by aciccarello** [comment]
@blakeembrey Do you have access to change the typedoc.org config to use HTTPS?

https://help.github.com/articles/troubleshooting-custom-domains/#https-errors

Thanks for the report @lgarron.

**Comment by jhpratt** [comment]
@blakeembrey Even if not enforced, defaulting to secure is always a good idea.

**Comment by blakeembrey** [comment]
@jhpratt I’m not sure I understand the comment, what is your expectation? I’m not sure there is a way to have HTTPS be the default without enforcing it.

**Comment by lgarron** [comment]
> I’m not sure there is a way to have HTTPS be the default without enforcing it.

Indeed, the "Enforce HTTPS" checkbox just redirects to HTTPS right now.
